### DISCUSSION TITLE
Drasil Research Group Meeting Agenda, 1:30 pm, Friday Feb 4, 2022 (On Teams)

### DISCUSSION THREAD

**Discussion by smiths** [JacquesCarette/Drasil#2929] [Meeting Agendas]
The agenda for this coming week is as follows:

1. Peter update
2. Dong update
3. Jason update
4. Cold cases:
a. Follow up to #1166 (Peter)
b. #2911 (Jason)
c. #2194 (Don)
d. #412 - At the very least, if we choose to not use pandoc, we might be able to learn from some of its [inner mechanics](https://hackage.haskell.org/package/pandoc-types-1.22.1/docs/src/Text.Pandoc.Definition.html#Pandoc)  (Jason)
e. #1715 (Peter)
5. Questions/Comments/Concerns/Discussion

**Comment by smiths** [comment]
- covered up to 4.b (inclusive)

**Reply by cd155** [reply]
Hi Dr. @smiths, do we have a meeting tomorrow? Will it be hosted in Teams or in-person?

**Reply by smiths** [reply]
Yes, we have a meeting.  It will be on Teams.
